src/styles/navbar.css

```diff
@@ -0,0 +1,47 @@
+.navbar__outer-div {
```

Comment:
As before the indentation should be brought in line with the rest of the project, and the extra blank lines should be removed from inside the CSS classes

If you wanted to tidy up a bit further, you could share some common properties by grouping them together, then just applying the properties to the individual classes as needed (IE `display: flex`, `flex-direction: row` and `align-items: center` are all used in `.navbar__outer-div`, `.navber__logo-title` and `navbar-links`), so these could be consolidated